gstd/src/lib.rs

```diff
@@ -28,6 +28,25 @@
 //! asynchronous programming primitives, arbitrary types encoding/decoding,
 //! providing convenient instruments for creating programs from programs, etc.
 //!
+//! # Crate features
+//!
+//! - **debug** - When enabled, logging becomes more verbose; this heavily
```

Comment:
Not the verbose, but in fact only then logging appears

Reply:
Fixed in 5e08210.